### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by eliottrobson** [henrikfroehling/Trakt.NET#623] [General]
I am using your package to support building an app I'm working on. I saw the `CONTRIBUTING.md` file but it was quite generic. What's your vision for the project? I am happy to help out with code changes or just feature requests depending if this is a personal passion project or not :)

**Comment by henrikfroehling** [comment]
Hi, the current plan is to get the next version 1.4.0 ([Milestone 1.4.0](https://github.com/henrikfroehling/Trakt.NET/milestone/8)) out. After that there might be some more 1.x versions for adding support for possible new API endpoints.

The next major version 2.0.0 will be a major refactoring and modernization, consisting of adding support for multiple .NET version targets, replacing the current Newtonsoft JSON implementation with the .NET System.Text.Json API and adding support for getting media (e.g. images) from other services like TMDb.
It's currently in an very early state and I've still to figure out what might be the best approach.

Version 1.4.0 is almost finished.
I'm currently updating the documentation for the upcoming version 1.4.0.

If you want, you can work on [your issue](https://github.com/henrikfroehling/Trakt.NET/issues/622) adding a new enum for "episode_type".
Feel free to ask any questions.
